CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This section contains changes that have been committed but not yet released.
 
 - Enabled support for Nylas API v2.2
 - Add `metadata` field in the Event model to support new event metadata
+- Add support for filtering `metadata` using `metadata_key`, `metadata_value`, and `metadata_pair`
+- Support new `message.bounced` webhook trigger
```

Comment:
This line splits the two related metadata lines, also likely due to the merge resolution.